e2e_test/bin/e2e_test.dart

```diff
@@ -10,8 +10,7 @@ const _exampleDsn =
 const _org = 'sentry-sdks';
 const _projectSlug = 'sentry-flutter';
 
-final _token =
```

Comment:
if this is the token from `SENTRY_AUTH_TOKEN`, lets rotate it, if its a personal one, rotate as well please

Reply:
Yeah, that was a personal